bindings/Makefile

```diff
@@ -16,8 +17,13 @@ deps/$(CODEGEN_BIN):
 build/%: $(SWAGGER_SPEC) deps/$(CODEGEN_BIN)
 	mkdir -p build/$*
 	java -jar deps/$(CODEGEN_BIN) generate -i $(SWAGGER_SPEC) \
-		-l $* -o build/$*
+		-l $* -o build/$* -c config.json
 	touch build/$*
+	if [ $* = "python" ] ; then \
+		cp -rf ./build/python/determined/_swagger/client/ $(CLIENT_PATH); \
```

Comment:
This looks like it works on BSD-systems but the -f behaves differently on linux systems, so there's still a bug here.